SPECS/libguestfs/libguestfs.spec

```diff
@@ -833,22 +825,23 @@ tdnf install --downloadonly -y --disablerepo=* \
     zfs-fuse \
 %endif
 
-
 mkdir cachedir repo
 find %{_var}/cache/tdnf -type f -name '*.rpm' -print0 | \
   xargs -0 -n 1 cp -t repo
 createrepo_c repo
 sed -e "s|@PWD@|$(pwd)|" %{SOURCE6} > yum.conf
+extra=--with-supermin-packager-config=$(pwd)/yum.conf
 
 %build
 # "--with-distro=REDHAT" is used to indicate Mariner is "Fedora-like" in package naming
 %configure \
   PYTHON=python3 \
   --with-default-backend=libvirt \
+  --enable-appliance-format-auto \
   --with-distro=REDHAT \
   --with-extra="release=%{release},libvirt" \
   --with-qemu="qemu-system-%{_build_arch} qemu" \
-  --with-supermin-packager-config=$(pwd)/yum.conf \
+  --with-extra="rhel=%{rhel},release=%{release},libvirt" \
```

Comment:
wondering if/why we need the `%{rhel}` part here?

Reply:
Oh, yeah, I was working with libvirt in testing, added it for testing purposes, removed it now